packages/noports_core/lib/src/common/default_args.dart

```diff
@@ -29,9 +29,9 @@ class DefaultSshnpArgs {
   static const List<String> localSshOptions = <String>[];
   static const bool legacyDaemon = false;
   static const bool listDevices = false;
-  static const SupportedSshClient sshClient = SupportedSshClient.exec;
+  static const SupportedSshClient sshClient = SupportedSshClient.openssh;
 }
```

Comment:
Renamed all "exec" ssh stuff to "openssh" so we can add support for other execution based ssh programs like dropbear and putty